.gitignore

```diff
@@ -1,3 +1,4 @@
 vendor/
 composer.lock
 phpunit.xml
+.idea/
```

Comment:
Please add it to your global .gitignore - https://help.github.com/articles/ignoring-files/#create-a-global-gitignore